mako/ext/babelplugin.py

```diff
@@ -54,5 +54,4 @@ def extract(fileobj, keywords, comment_tags, options):
     :rtype: ``iterator``
     """
     extractor = BabelMakoExtractor(keywords, comment_tags, options)
-    for message in extractor(fileobj):
-        yield message
+    yield from extractor(fileobj)
```

Comment:
I think mako still supports py2.7.

@zzzeek should this ho in 1.2 that removes py2?

this applies to all other `yield from`